unpackaged/config/platformencryption_fields/objects/Contact.object

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@jofsky and we'll have to add the fields/objects we want to encrypt to this directory? If this works, that is going to be friggin' awesome.